WordPress/Classes/Services/XpostSuggestionService.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+
+/// A service to fetch and persist a list of sites that can be used for x-posting.
+struct XpostSuggestionService {
```

Comment:
`XpostSuggestionService` should probably live in https://github.com/wordpress-mobile/WordPressKit-iOS — however, since `SuggestionService` was already implemented here in WordPress-iOS I added it here as well.

I'm open to suggestions though. At the moment my plan is to move both `SuggestionService` and `XpostSuggestionService` to WordPressKit at a later stage (created issue https://github.com/wordpress-mobile/WordPress-iOS/issues/15146).